### PR TITLE
{chem}[foss/2021b] Amber v22.0 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/c2d86642aafe6c0ad5a13930cf7f0678 for a full test report.

File: easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+name = 'Amber'
+local_amber_ver = 22
+local_ambertools_ver = 22
+# Patch levels from http://ambermd.org/AmberPatches.php and http://ambermd.org/ATPatches.php
```
Comment (bedroge) [inline easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb:4]:
```suggestion
# Patch levels from https://ambermd.org/AmberPatches.php and https://ambermd.org/ATPatches.php
```

File: easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+patchlevels = (3, 0)  # (AmberTools, Amber)
+version = '%s.%s' % (local_amber_ver, patchlevels[1])
+versionsuffix = '-AmberTools-%s.%s' % (local_ambertools_ver, patchlevels[0])
+
+homepage = 'http://ambermd.org/amber.html'
```
Comment (bedroge) [inline easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb:9]:
```suggestion
homepage = 'https://ambermd.org/amber.html'
```

**Review by bedroge — CHANGES_REQUESTED** [review]
Looks like https can be used for all URLs (also see #16109); I've just done the same for #16123.

File: easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb (hunk 2 above)
Comment (akesandgren) [reply easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb]: Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/16166/commits/eef360dc8d0f30d27748cef9452087f11dcbaf4a

File: easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    {'name': 'AmberTools-21_fix_incorrect_dvout_call.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-21_fix_incorrect_mexit_calls.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-21_fix_more_blas_argument_problems.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-21_fix_multiple_definition.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-21_fix_potential_use_before_init.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-21_fix_rism_argument_mismatch.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-21_fix_xray_fftpack_arg_mismatch.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-22_fix_test_missing_cuda_dir.patch', 'alt_location': 'AmberTools'},
+    {'name': 'AmberTools-22_fix_missing_error_check_on_test_run.patch', 'alt_location': 'AmberTools'},
+    'Amber-22_reduce_precision_of_kmmd_test.patch',
```
Comment (bedroge) [inline easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb:35]: These two patches still seem to be missing?

Comment (akesandgren) [reply easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb:35]: Oh did I forget them... fixing.

Comment (akesandgren) [reply easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb:35]: And the AmberTools patch is pending #16136

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/4d81ea0032e2a2713219a91c82d5035f for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/56d2a0d3a15cc1d4e3407bb33272356a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**